lms/templates/footer.html

```diff
@@ -83,3 +83,19 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+% if settings.FEATURES.get('ENABLE_COOKIE_CONSENT', False):
+<!-- Cookie Consent -->
```

Comment:
Note also that we indent content inside ``if`` statements so that the nesting is clear.